src/libsystemd-network/test-ipv4ll-manual.c

```diff
@@ -96,12 +103,14 @@ int main(int argc, char *argv[]) {
         test_setup_logging(LOG_DEBUG);
 
         if (argc == 2)
-                return test_ll(argv[1], NULL);
+                return test_ll(argv[1], NULL, NULL);
         else if (argc == 3)
-                return test_ll(argv[1], argv[2]);
+                return test_ll(argv[1], argv[2], NULL);
+        else if (argc == 4)
+                return test_ll(argv[1], argv[2], argv[3]);
         else {
-                log_error("This program takes one or two arguments.\n"
-                          "\t %s <ifname> [<seed>]", program_invocation_short_name);
+                log_error("This program takes one, two or three arguments.\n"
+                          "\t %s <ifname> [<seed>] [<start_address>]", program_invocation_short_name);
```

Comment:
When the starting address is specified, the seed will be ignored. Please make the second argument is a seed or address, i.e. `"%s ifname [seed|address]"`.